cabal.project

```diff
@@ -10,8 +10,8 @@ repository cardano-haskell-packages
     d4a35cd3121aa00d18544bb0ac01c3e1691d618f462c46129271bccf39f7e8ee
 
 index-state:
-  , hackage.haskell.org 2024-10-10T00:52:24Z
-  , cardano-haskell-packages 2024-11-26T16:00:26Z
+  , hackage.haskell.org 2025-05-23T06:30:40Z
+  , cardano-haskell-packages 2025-05-16T20:03:45Z
```

Comment:
Curious how was this index state chosen? Does it follow a node version?

Reply:
I do not see why it has to follow the `node`.